packages/components/filter-bar/_filter-bar-theme.scss

```
@@ -0,0 +1,61 @@
+@use '../core/styles/common/tokens' as *;
+
+@mixin kbq-filter-bar-typography() {
+    .kbq-hint.kbq-date-period__hint .kbq-hint__text {
+        @include kbq-typography-level-to-styles-css-variables(typography, text-compact);
```

```suggestion
        @include tokens.kbq-typography-level-to-styles-css-variables(typography, text-compact);
```

предлагаю не использовать `as *` при импорте

---

почему ? если файл небольшой и к примеру всего пару @use почему бы нет